common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -343,6 +343,9 @@ else if (isRedirectUrl(formattedURL)) {
                 processWebCpEnrollmentUrl(view, url);
             } else if (mIsWebCpInWebViewFeatureEnabled && isWebCpAuthorizeUrl(url)) {
                 processWebCpAuthorize(view, url);
+            } else if (isDeviceCaRequest(url) && isHttpsScheme(url)) {
```

Comment:
What if it's deviceCaRequest true but NOT https scheme? That's the general case, right? Where is that getting handled now?

Reply:
Even before we reach the if condition I added
<img width="717" height="113" alt="image" src="https://github.com/user-attachments/assets/e76c2596-c821-44c5-96a0-36a27406cd67" />